### PR TITLE
Expand functionality of the reddit handler

### PR DESCRIPTION
**Pull request by s7904893**

### CODE REVIEW

**Comment by fried-ice** [comment]
The current solution adds a button with a link, that opens the image in a webbrowser.

Is there a way to load the image directly in the app?

**Comment by fried-ice** [comment]
Also, the inline feature (`@Bot`)  seems to be broken.

**Comment by s7904893** [comment]
> The current solution adds a button with a link, that opens the image in a webbrowser.
> 
> Is there a way to load the image directly in the app?

I tried some things but it doesn't seem there is a solution for this. You can upload the image as a Document (via send_document) with the blurred image as a thumbnail, but as soon as you specify the file format as an image format, the thumnail gets ignored and the full image is shown.

File: src/main.py [hunk 1]
```diff
@@ -166,33 +181,70 @@ def decision(update, context):
...
+                    posts_sent = True
+                elif post_type == RedditPostTypes.animation:
+                    for site in REDDIT_EXCLUDED_ANIMATION_SITES:
+                        if site in post.url:
+                            pass
+                    context.bot.send_animation(chat_id=update.message.chat_id, animation=post.url, caption=post.title)
+                    posts_sent = True
+
+    except Exception as e:
+        print(e)
```
Comment (fried-ice) [inline src/main.py:242]: hmmm.
guess we can leave that for now.

File: src/main.py [hunk 2]
```diff
@@ -166,33 +181,70 @@ def decision(update, context):
...
-        for ending in imageFileEndings:
-            if str(post.url).endswith(ending):
-                images.append(post.url)
+        if get_post_type(post) == RedditPostTypes.image:
+            images.append(post.url)
     return images
 
 
-def sendSubredditImages(subreddit_display_name, update, context):
+def send_subreddit_posts(subreddit, update, context, offset=0, count=5):
```
Comment (fried-ice) [inline src/main.py:213]: For the future, we should use the same code for the posting list iteration for both `send_subreddit_posts` and `get_subreddit_images`

**Review by fried-ice — APPROVED** [review]
For the future, please don't switch naming conventions in a feature commit, open a new issue and create a branch just for that matter.

**Pull request merged**